webapp/medunited/care/view/medication/Master.view.xml

```diff
@@ -43,8 +43,7 @@
 					</Column>
 				</columns>
 				<items>
-					<ColumnListItem type="Navigation"
-						press="onListItemPress">
+					<ColumnListItem type="Selection" press="onListItemPress">
```

Comment:
I don't think that type "Selection" exists. You can leave this just with ColumnListItem, the rest is not needed anymore.